FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -111,6 +111,17 @@
  */
 @property (readwrite, nonatomic, assign) FBSnapshotTestCaseAgnosticOption agnosticOptions;
 
+/**
+ When set, resets the test name to the specified one, which in turn also resets the folder in
+ which the snapshopt is going to be saved.
+ */
```

Comment:
One big problem with adding a `testName` property; if you set it after `[super setUp]` it will still work but only until `FBSnapshotVerifyView()` is called. But `FBSnapshotTestController` was written assuming that it would get its `testName` through its initializer. Your headerdoc should mention that `testName` can only be set after `[super setUp]`.

Reply:
Also add an assert like we use for other properties:

```
  NSAssert1(_snapshotController, @"%s cannot be called before [super setUp]", __FUNCTION__);
```